packages/tailwindcss-config/tailwind.config.ts

```diff
@@ -6,6 +6,20 @@ const config: TailwindConfig = {
   content: [],
   theme: {
     extend: {
+      keyframes: {
+        fadeInLeft: {
+          "0%": { opacity: "0", transform: "translateX(-100%)" },
+          "100%": { opacity: "1", transform: "translateX(0)" },
+        },
+        fadeOutLeft: {
+          "0%": { opacity: "1", transform: "translateX(0)" },
+          "100%": { opacity: "0", transform: "translateX(-100%)" },
+        },
+      },
+      animation: {
+        fadeInLeft: "fadeInLeft 0.5s ease-in-out",
+        fadeOutLeft: "fadeOutLeft 0.5s ease-in-out",
+      },
```

Comment:
pondría los nombres de las animaciones separados con guiones (tipo "fade-in-left", entre comillas)